### ISSUE TITLE
Tooltip not displayed in Workspace name 

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#5907]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

Issue is failing PR https://github.com/Expensify/App/pull/5812

## Action Performed:
1. Navigate to workspace settings
2. Hover over the workspace name

## Expected Result:
Tooltip should appear with the full workspace name 

## Actual Result:
Tooltip does not appear. 

## Workaround:
None needed, visual issue. 

## Platform:

Where is this issue occurring?

- Web
- Desktop App

**Version Number:** 1.1.8-0

**Reproducible in staging?:** Yes 
**Reproducible in production?:** Yes (but should be fixed in PR https://github.com/Expensify/App/pull/5812)

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation
![image](https://user-images.githubusercontent.com/44479856/137566497-05021dcf-3429-4aed-a6c2-0ae6acd7d977.png)

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by akshayasalvi** [comment]
It is working fine in the `main` branch. 

<img width="424" alt="Screenshot 2021-10-16 at 4 00 23 PM" src="https://user-images.githubusercontent.com/57435789/137583990-625ecc28-7a6b-417c-a9a8-9d5d0082ab67.png">
<img width="409" alt="Screenshot 2021-10-16 at 4 00 11 PM" src="https://user-images.githubusercontent.com/57435789/137583993-e76c57c8-9cde-45e9-8c90-f482e02a5fa0.png">

**Comment by tgolen** [comment]
I agree that it looks like the code wasn't deployed properly. I don't think this should be a deploy blocker so I am demoting the issue and my suggestion is to re-evaluate this after the next deploy to staging. If it's still not there, then we'll need to figure out how to fix it properly for staging.

**Comment by Beamanator** [comment]
The part that baffles me is that the "Added tooltip for workspace name" actually does seem to exist in staging (see screenshot) but only the tooltip import!:

![Screen Shot 2021-10-18 at 2 36 22 PM](https://user-images.githubusercontent.com/3885503/137803145-02285709-d065-4cc0-ab4d-a40fae22714d.png)

**Comment by Beamanator** [comment]
Clicking "View at this point in history" on the "Added tooltip for workspace name" commit on [this page of historical commits](https://github.com/Expensify/App/commits/staging/src/pages/workspace/WorkspaceInitialPage.js), you can see the entire commit exists (importing Tooltip & using the Tooltip).

There's only one more commit on that file in the history (see link above), which is https://github.com/Expensify/App/commit/92953074b7cbe381fa8a0cc7fd933e0af5589d37#diff-f4168366e38bc17178db9db7be5a28974510a61b5b657dcfe9cc86ddc2b51677, a "merge commit" -> So I think that one mucked up somehow

**Comment by roryabraham** [comment]
I don't think so ... but great investigation so far! My _guess_ is that someone had to manually resolve conflicts when a PR was CP'd to staging, and accidentally got rid of the tooltip.

**Comment by roryabraham** [comment]
Wait a minute ... how can I submit a PR if the code's already on main? 🙃 Need a new plan 😆

**Comment by tgolen** [comment]
woohoo! Thanks, Rory.
